### DISCUSSION TITLE
Replication between 2 OrientDB over SSL

### DISCUSSION THREAD

**Discussion by namlegief** [orientechnologies/orientdb#9765] [General]
Hi , I’m trying to understand if it possible to replicate between 2 OrientDB servers over SSL . I can replicate with hazelcast , but as I understand it not secure . 
please advice .

**Comment by timw** [comment]
Using Hazelcast with TLS requires configuring Hazelcast itself, and is addressed in the Hazelcast docs [1], and configured in `config/hazelcast.xml`.

Note that the actual data replication is done directly between OrientDB nodes without going through Hazelcast, so you probably also want to configure TLS on the OrientDB binary port if you haven't already [2].

[1] : https://docs.hazelcast.com/imdg/3.12/security/tls-ssl#tlsssl-for-hazelcast-members
[2]: http://orientdb.com/docs/3.1.x/security/Using-SSL-with-OrientDB.html
